client-code-gen/README.md

```diff
@@ -0,0 +1,52 @@
+# FAM API Client Code Generation for Frontend with "typescript-axios"
+
+This folder contains a mini npm project for generating FAM API client from OpenAPI(Swagger) spec. The genereated client (library) can be utilized for frontend with typescript/axio framework to make api call to FAM backend endpoints.
+
+Client library regeneration is needed when endpints contract has changed.
+
+## Package/Tool choice
+This is using the popular [OpenAPI Generator/openapi-generator-cli](https://openapi-generator.tech/docs/installation) for generation.
+However, it requires java installed in the environment.
```

Comment:
We should have a Dockerfile to do this generation that contains the java environment so we don't need to install java locally and can guarantee the java version used.

Reply:
As per discussion will do separate PR for the dockerfile.